tests/fsharp/conformance/lexicalanalysis/E_LessThanDotOpenParen001.bsl

```diff
@@ -0,0 +1,48 @@
+
+E_LessThanDotOpenParen001.fsx(13,71,13,75): typecheck error FS0193: A type parameter is missing a constraint 'when ( ^T or  ^?8151) : (static member ( + ) :  ^T *  ^?8151 ->  ^?8152)'
```

Comment:
Seeing this test failing for first time on CI while it passes locally.

> (static member ( + ) :  ^T *  ^?8151 ->  ^?8152)'

I will try again in a release build if that somehow changes the result and is consistent with CI, it used to pass so I'm checking if rebasing on master has any effect.

Originally, this test was not asserting on the whole output and was in fsharpqa, asserting only on a subset of the errors.

It is related to this tracked issue: https://github.com/dotnet/fsharp/issues/6725 and the test will keep showing jumping values with changes done in the compiler.

The error in question is unrelated to the changes in this PR.